ceph/ceph_admin/__init__.py

```diff
@@ -124,6 +124,15 @@ def set_tool_repo(self, repo=None):
             for node in self.cluster.get_nodes():
                 node.exec_command(sudo=True, cmd=cmd)
 
+    def set_cdn_tool_repo(self):
+        """
+        Add the cdn Tools repo on nodes.
+        """
+        cdn_repo = "rhceph-5-tools-for-rhel-8-x86_64-rpms"
```

Comment:
the tools repo for 5 is in the `ceph.py` python file as well, can we use it from ceph package?